fortanix-sgx-tools/src/bin/ftxsgx-runner-cargo.rs

```diff
@@ -29,12 +29,12 @@ struct Target {
 #[derive(Deserialize, Debug)]
 #[serde(rename_all = "kebab-case")]
 struct Metadata {
-    fortanix_sgx: Target
+    fortanix_sgx: Option<Target>
 }
 
 #[derive(Deserialize, Debug)]
 struct Package {
-    metadata: Metadata
+    metadata: Option<Metadata>
```

Comment:
Instead of using `Option`, add `#[derive(Default)]` on `Metadata` and `Target` and `#[serde(default)]` on the fields in `Package` and `Metadata`. This simplifies the code below.

Reply:
Done